CONTRIBUTORS.md

```diff
@@ -7,4 +7,7 @@ GitHub contributors:
 - Progi1984
 - Rémi Gaillard
 - Thomas N
-- atomiix
\ No newline at end of file
+- atomiix
+- Mathieu Ferment
+- Jonathan Danse
+- Vincent Garcia
```

Comment:
Should we use nicknames instead?

Reply:
It's seems that it's the name shown into the profile, instead of the author login, according to the previous change and the PrestaShop\PrestaShop repo.

For me, the best, is to have something as:

- NeOMakinG (SZCZUPAK Valentin) ;
- SZCZUPAK Valentin (NeOMakinG)

Reply:
Agree about this, but for the moment, let's stick to what we use everywhere